include/doap-geenrator.html

```diff
@@ -0,0 +1,27 @@
+<html>
+    <head>
+        <title>DOAP Generator</title>
```

Comment:
filename should be "doap-generator.html"

Reply:
This html document is superfluous now so have removed